src/replication.c

```diff
@@ -190,6 +198,77 @@ void rebaseReplicationBuffer(long long base_repl_offset) {
     }
 }
 
+/* Replication: Primary side - connections association.
+ * On rdb-connection sync, connection association is used to keep replication data in
+ * the backlog until the replica requests PSYNC. Association happens in two forms,
+ * if there's an existing buffer block at the fork time, the replica is attached
+ * to the tail, if there is no tail, the replica will be attached when a new
+ * buffer block is created (see the Retrospect function below).
+ * Replica rdb client id is used as a unique key for the association.
+ * On COB overrun, association is deleted and the RDB connection
+ * is dropped.
+ */
+void addReplicaToPsyncWait(client *replica) {
```

Comment:
I think we should qualify this replica with `rdb_` to disambiguate from the replica used in `removeReplicaFromPsyncWait`
```suggestion
void addRdbReplicaToPsyncWait(client *rdb_replica) {
```

Reply:
Its a bit confusing cause it sounds like there are two replicas.
What about `replica_rdb_client`and `replica_main_client`?

Reply:
Yep - I like them